src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -711,7 +718,10 @@ def create(self, vol_name: str, subvol_name: str, **kwargs):
 
         return f'Subvolume {subvol_name} created successfully'
 
-    def set(self, vol_name: str, subvol_name: str, size: str):
+    def set(self, vol_name: str, subvol_name: str, size: str, group_name: str = ""):
+        params = {'vol_name': vol_name, 'sub_name': subvol_name}
+        if group_name:
+            params['group_name'] = group_name
```

Comment:
the `params` is not used with the call `_cmd_fs_subvolume_resize`, and hence the `Edit` fails.